### PR TITLE
[doc] remove references to AIR

### PR DESCRIPTION
**Pull request by richardliaw**
## Why are these changes needed?

Reduce references to AIR. Need to change some examples to get rid of more references.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by pcmoritz** [comment]
We also need to get rid of all references to AIR sessions in the docs and replace them with `train.report`, `train.get_dataset_shard` and train contexts :)

File: README.rst [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
-Ray is a unified framework for scaling AI and Python applications. Ray consists of a core distributed runtime and a toolkit of libraries (Ray AIR) for simplifying ML compute:
+Ray is a unified framework for scaling AI and Python applications. Ray consists of a core distributed runtime and a set of AI libraries for simplifying ML compute:
 
 .. image:: https://github.com/ray-project/ray/raw/master/doc/source/images/what-is-ray-padded.svg
 
 ..
   https://docs.google.com/drawings/d/1Pl8aCYOsZCo61cmp57c7Sja6HhIygGCvSZLi_AuBuqo/edit
 
-Learn more about `Ray AIR`_ and its libraries:
+Learn more about `Ray AI Libraries`_:
```
Comment (jjyao) [inline README.rst:24]: Is AIR still a term we should use when talking to users?

I thought we just remove AIR as a concrete package but still keep the AIR term to represent the collection of AI libraries?

**Review by ericl — APPROVED** [review]
Stamped.

File: README.rst (hunk 1 above)
Comment (ericl) [reply README.rst:24]: I think we should refer to them as AI libraries generally (@richardliaw I really like this proposal, don't know why we went with AIR originally--- probably too catchy sounding).

Anecdotally, most users are thinking library centric anyways, so the term "AIR" rarely comes up organically.

**Review by matthewdeng — APPROVED** [review]

**Review by justinvyu — APPROVED** [review]

**Pull request merged**